src/js/vic-v2/definitions/address.js

```diff
@@ -2,10 +2,34 @@ import _ from 'lodash/fp';
 import { createSelector } from 'reselect';
 
 import { countryLabels, countryValues, states } from '../labels';
-import { validatePostalCodes } from '../../common/schemaform/validation';
 
 const requiredFields = ['street', 'city', 'country', 'state', 'postalCode'];
 
+export function isValidUSZipCode(value) {
```

Comment:
It looks like this is a duplicate of the one in `src/js/common/utils/address.js`, is this another optimization?

Reply:
Yeah, I don't want to import that file because it has labels that this address field doesn't need. It felt more awkward to make another file just for these zip code functions.